pkg/controllers/dynakube/controller_test.go

```diff
@@ -352,14 +356,15 @@ func TestReconcileComponents(t *testing.T) {
 			injectionReconcilerBuilder:  createInjectionReconcilerBuilder(mockInjectionReconciler),
 			oneAgentReconcilerBuilder:   createOneAgentReconcilerBuilder(mockOneAgentReconciler),
 			logModuleReconcilerBuilder:  createLogModuleReconcilerBuilder(mockLogModuleReconciler),
+			extensionReconcilerBuilder:  createExtensionReconcilerBuilder(mockExtensionReconciler),
 		}
 		mockedDtc := dtclientmock.NewClient(t)
 
 		err := controller.reconcileComponents(ctx, mockedDtc, nil, dk)
 
 		require.Error(t, err)
 		// goerrors.Join concats errors with \n
-		assert.Len(t, strings.Split(err.Error(), "\n"), 4) // ActiveGate, OneAgent LogModule, and Injection reconcilers
+		assert.Len(t, strings.Split(err.Error(), "\n"), 5) // ActiveGate, OneAgent LogModule, and Injection reconcilers
```

Comment:
Didn't update the comment 😉

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/3875/commits/1c4abfe681211d132ec48698079d2618271bdec9